app/build.gradle

```diff
@@ -9,9 +9,8 @@ android {
         applicationId "org.literacyapp"
         minSdkVersion 21
         targetSdkVersion 23
-        versionCode 1003008
-        versionName "1.3.8"
-        setProperty("archivesBaseName", "$applicationId-$versionCode")
```

Comment:
No, it caused some problems. When I made code changes and deployed the APK via Android Studio, the latest code changes were not picked up since it picked the wrong APK file.